src/main/java/sh/platform/config/Credential.java

```diff
@@ -1,12 +1,15 @@
-package sh.platform.config.reader;
+package sh.platform.config;
 
 import java.util.Collections;
 import java.util.Map;
 import java.util.Optional;
 
+/**
+ * An Unit  of a service that comes from {@link PlatformVariables#PLATFORM_RELATIONSHIPS}
+ */
 public class Credential {
 
-    private final Map<String, Object> config;
+    protected final Map<String, Object> config;
```

Comment:
I am not sure what the Java conventions are here, but I am always a fan of protected over private.  I may be in the minority, though. :smile:

Reply:
In Java, private is always comes first:
Minimize the accessibility of classes and members from Effective Java :)
I changed because I need this field on format implementations.